pipelines/basic/scripts/update-version.sh

```diff
@@ -73,14 +73,15 @@ else
 fi
 
 # Update changelog
+release_notes_template="##### Breaking Changes\n* None.\n\n##### New Features\n* None.\n\n##### Enhancements\n* None.\n\n##### Fixes\n* None.\n\n##### Notes\n* None.\n"
```

Comment:
What's the reason behind going down two levels of headings instead of one? Is it just easier to read that way?

Reply:
I would like it if we had consistent formatting and styling accross all of our products and libraries

Reply:
getting there... this brings us one (small) step closer